src/qibolab/_core/components/configs.py

```diff
@@ -43,6 +46,22 @@ class DcConfig(Config):
 
     offset: float
     """DC offset/bias of the channel."""
+    filters: Filters = Field(default_factory=list)
+    """List of filters."""
+
+    @property
+    def feedback(self) -> list[float]:
+        feedback_coeff = [i.feedback for i in self.filters if i is not None]
+        if len(feedback_coeff) > 0:
+            return reduce(np.convolve, feedback_coeff)
+        return []
```

Comment:
In this case, you should be able to safely replace this with:
```suggestion
        return reduce(np.convolve, feedback_coeff, [1])
```
since `[1]` is the neutral element of the convolution.

There should be no side effect in applying a `[1]` filter, against not doing anything (possibly an overall delay of the experiment - but it is pretty irrelevant).
In case there are, or you really want to avoid it, I would suggest just reversing the condition, in the spirit of *early return*:


```suggestion
        if len(feedback_coeff) == 0:
            return []
        return reduce(np.convolve, feedback_coeff)
```